### PR TITLE
Added 2 endpoints

### PR DESCRIPTION
**Pull request by samasno**
## Trello
https://trello.com/c/pZA2h029/32-configure-first-iteration-of-review-service-api

## Setup
In root directory:
rename `example.env` to `.env` if first time running service
 `npm i`
`npm run seed`

## Run Server
From project direcotry  `npm run dev`

## Use
Run Server
In browser navigate or in  Postman make GET request to:
`http://localhost:5002/reviews-api/info/1` (1-100 valid)
 `http://localhost:5002/reviews-api/all/1` (1 - 100 valid)

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1,3 +1,4 @@
 MONGO_URI_DEV=mongodb://localhost:27017/review-service-dev
 MONGO_URI_PROD=mongodb://localhost:27017/review-service-prod
-MONGO_URI_TEST=mongodb://localhost:27017/review-service-test
\ No newline at end of file
+MONGO_URI_TEST=mongodb://localhost:27017/review-service-test
+PORT=
```
Comment (christulin) [inline example.env]: this PORT needs to be filled out for server to properly connect, thinking you might want to fill this in with `5002`?

File: server/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    let err = new Error(errMess);
+    err.status = status;
+    throw err;
+  }
+}
+
+//GET @  /reviews-api/info/:workspace-id 
+exports.reviewInfo = ah(async (req, res, next) => {
+  const { workspaceId } = req.params;
+  checkExists(workspaceId, 'No valid ID', 400);
```
Comment (christulin) [inline server/controllers/index.js]: I'm getting a 'Path Does Not Exists' error message when I leave out ID from `notFound`, so I'm not sure there is ever a US case where server responds with `400`..  I don't think this first `checkExists` is actually ever used currently.  Could be good opportunity to write some tests.

**Review by christulin — DISMISSED** [review]

File: server/controllers/index.js (hunk 2 above)
Comment (samasno) [reply server/controllers/index.js]: You're right, it'll never get a hit. Will remove the first calls

File: example.env (hunk 1 above)
Comment (samasno) [reply example.env]: Will add it

**Review by joshuabradley012 — APPROVED** [review]

**Pull request merged**